src/uct/api/v2/uct_v2.h

```diff
@@ -530,21 +536,26 @@ ucs_status_t uct_md_mem_dereg_v2(uct_md_h md,
 /**
  * @ingroup UCT_MD
  *
- * @brief Pack a remote key.
+ * @brief Pack a memory key as a remote or shared one.
+ *
+ * This routine packs a local memory handle registered by @ref uct_md_mem_reg
+ * into a memory buffer which then could be deserialized by a peer and used
+ * in UCP operations. 
  *
  * @param [in]  md          Handle to memory domain.
  * @param [in]  memh        Pack a remote key for this memory handle.
  * @param [in]  params      Operation parameters, see @ref
  *                          uct_md_mkey_pack_params_t.
- * @param [out] rkey_buffer Pointer to a buffer to hold the packed remote key.
+ * @param [out] buffer      Pointer to a buffer to hold the packed remote key.
  *                          The size of this buffer has should be at least
```

Comment:
plz remove `has` (it's an old code, but you already modifying it)

Reply:
done